cg-spec/editors_draft.html

```diff
@@ -3031,11 +3033,101 @@ <h2>Alternatives to referential opacity</h2>
           </li>
         </ul>
 
+        <div class=note>Other design choices for the semantics have also been discussed by the community group; these have been <a href="https://github.com/w3c/rdf-star/issues/95">summarized in a Github issue</a>.</div>
+
       </section>
 
-    </section>
+      <section id="selective-ref-transparency">
+        <h2>Selective referential transparency</h2>
+
+        <p>Since the need for referential opacity/transparency is use-case dependent (as discussed <a href="#ref-opacity">above</a>), it may make sense for different predicates to have different behaviors. Consider the following example.</p>
```

Comment:
```suggestion
        <p>Since the needs for referential opacity and transparency are use-case dependent (as discussed <a href="#ref-opacity">above</a>), it may make sense for different predicates to have different behaviors. Consider the following example.</p>
```